### DISCUSSION TITLE
Package installment suggestion

### DISCUSSION THREAD

**Discussion by mengyuan404** [PaddlePaddle/PaddleSpatial#209] [BaiduKDDCup2022] [answered]
Could you install the following packages in the pytorch and paddle environment?
tsfel, featuretools, statsmodels, xgboost, lightgbm, DGL, PYG, Sktime, mlxtend, tsfresh,

**Comment by ramber1836** [comment, marked as answer]
Most of the requested pkgs have been installed. 
The package sktime is incompatible with the current paddlepaddle environment.
